sdk/containerregistry/ci.yml

```diff
@@ -40,4 +40,5 @@ extends:
     - name: azure-mgmt-containerregistry
       safeName: azuremgmtcontainerregistry
     - name: azure-containerregistry
-      safeName: azurecontainerregistry
\ No newline at end of file
+      safeName: azurecontainerregistry
+    VerifyAutorest: false
```

Comment:
In these the udpated ci.yml, `VerifyAutorest` is set to `false`. Are these ones that you KNOW will have issues, so you're kinda pushing it off? That's totally fine, but what about the rest of the repo? Have you run `VerifyAutorest` against all the service directories?

Trying to get a feel for the impact if we check this in. 

Also, have you let the mgmt folks know about this new check?

Reply:
I know this will cause issues with containerregistry and storage from testing it on both. I will reach out to the storage team to see if they want me to default it on. I have not let mgmt team know about it, but will reach out.